packages/devtools/src/launcher.js

```diff
@@ -60,10 +62,13 @@ async function launchChrome (capabilities) {
     }
 
     log.info(`Launch Google Chrome with flags: ${chromeFlags.join(' ')}`)
-    const chrome = await launchChromeBrowser({
+
+    const chrome = await launchChromeBrowser(Object.assign({
```

Comment:
We still don't need `Object.assign` here if `ignoreDefaultArgs` can contain the same values as in Puppeteer we can just do:

```js
const chrome = await launchChromeBrowser({
    chromePath: chromeOptions.binary,
    ignoreDefaultArgs,
    chromeFlags
})
```